cloudify_cli/tests/commands/test_plugins.py

```diff
@@ -116,3 +120,109 @@ def test_plugins_upload_with_visibility(self):
         yaml_path = os.path.join(PLUGINS_DIR, 'plugin.yaml')
         self.invoke('cfy plugins upload {0} -l private -y {1}'
                     .format(yaml_path, yaml_path))
+
+
+class PluginsUpdateTest(CliCommandTest):
+
+    def _mock_wait_for_executions(self, value):
+        patcher = patch(
+            'cloudify_cli.execution_events_fetcher.wait_for_execution',
+            MagicMock(return_value=PropertyMock(error=value))
+        )
+        self.addCleanup(patcher.stop)
+        patcher.start()
+
+    def setUp(self):
+        super(PluginsUpdateTest, self).setUp()
+        self.use_manager()
+        self.client.executions = MagicMock()
+        self.client.plugins_update = MagicMock()
+        self._mock_wait_for_executions(False)
+
+    def test_plugins_get(self):
```

Comment:
what about getting not valid id

Reply:
That should be the click option responsibility since the IDs here have the same rules for all resourced.

Reply:
Not sure about that

Reply:
https://github.com/cloudify-cosmo/cloudify-cli/blob/master/cloudify_cli/cli/cfy.py there's a validation function there. I think I should add the plugins_update_id arg option to the options there. Same for the deployment update.

Reply:
Test what happen if get not existing dep id